src/pb_stub.cc

```diff
@@ -771,6 +769,8 @@ main(int argc, char** argv)
         }
       });
 
+  stub->Initialize(model_version, argv[6] /* triton install path */);
```

Comment:
Add comment about why initialization needs to be delayed till here

Reply:
Done